ranger-gcs-plugin/README.md

```diff
@@ -0,0 +1,8 @@
+# Ranger GCS Plugin
```

Comment:
No need to mention the plugin, just describe what this script is: blocking non-root user access to the metadata server.

Reply:
This is about this folder. There may be other scripts here in the future

Reply:
I don't think we should have an "umbrella" folder in the init action repo. The script should just be about what it does, in this case, blocking non-root user access to metadata server.

If we have another init action script that performs a different job which might be used for the plugin, just create a new folder for that job.

Reply:
The repo is sturctured into components, this may not fit the structure well